components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -304,12 +303,11 @@ void AtmosphereOutput::register_views()
   // Cycle through all fields and register.
   for (auto const& name : m_fields)
   {
-    auto fmap = m_field_repo->get_field(name, m_grid_name);
+    auto field = m_field_repo->get_field(name, m_grid_name);
     // If the "averaging type" is instant then just need a ptr to the view.
-    auto view_d = fmap.get_view();
+    auto view_d = field.get_view();
     // Create a local copy of view to be stored by output stream.
-    auto view_copy_h = Kokkos::create_mirror_view( view_d );
-    view_type view_copy("",view_copy_h.extent(0));
+    view_type view_copy("",view_d.extent(0));
```

Comment:
I would advocate for using `field.get_view<Host>()` directly here (rather than creating view_copy), unless there is a reason to use a completely separate view.

Reply:
Also, do you _need_ the deep copy to host here? I would argue the deep copy is to be done only when the output is about to be written, not during registration...

Reply:
We need a separate view because this keeps track of "averaging", "min" and "max" type outputs too.  So we need to update a view that is independent of the field view.

Reply:
per deep_copy.  That is a good point.  We could probably keep the l_view that we track on device.  I'll work on that.

Reply:
I looked at the deep_copy, and I think we need to keep it.  Again, because the "view_copy" in this case refers to a separate view that will be used for output.  I suppose we could keep the view on device and just do a deep copy right before writing the output.